vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/test/grid-connector-cell-focus.test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, fixtureSync, nextFrame } from '@open-wc/testing';
+import { init, setRootItems, getBodyCell } from './shared.js';
+import type { FlowGrid } from './shared.js';
+import { GridColumn } from '@vaadin/grid';
+import sinon from 'sinon';
+
+describe('grid connector - cell focus', () => {
+  let grid: FlowGrid;
+  const columnFlowId = 'col0';
+
+  beforeEach(async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column path="name"></vaadin-grid-column>
+      </vaadin-grid>
+    `);
+
+    const keyColumn = grid.querySelector<GridColumn & { _flowId: string }>('[path="name"]')!;
+    keyColumn._flowId = columnFlowId;
+
+    init(grid);
+    setRootItems(grid.$connector, [{ key: '0', name: 'foo' }]);
+    await nextFrame();
+  });
+
+  it('should dispatch a grid-cell-focus event on cell focus', async () => {
```

Comment:
Should this check be covered as well?

https://github.com/vaadin/flow-components/blob/182a875769714107f619d5ef4cfabb31a6c57f1e/vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js#L1137-L1139

Reply:
Added the test case